Enhancing SSO instructions (#9780)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/set-up-sso-microsoft-entra-id.md

7. After registering the new application without specifying a Redirect URI, click on **App registration** and then navigate to the **Authentication** tab for the new application.

8. Click **+ Add platform** and enter a Redirect URI for your application. See step 4 above for more information on the correct Redirect URI value for your <Constant name="dbt" /> application.
    
    :::info Platform type

    When selecting the platform type, choose **Web**, not **Single-page application (SPA)**. The <Constant name="dbt" /> SSO integration redeems the authorization code from the server using a client secret. So, if you add the **Redirect URI** under **SPA**, Entra ID enforces PKCE and rejects the server-side token exchange, causing sign-in to fail with the error `AADSTS9002325: Proof Key for Code Exchange is required for cross-origin authorization code redemption.`

> 

Reviewer: Hiya Matt

The Web vs. SPA callout is currently only included in the alternate Authentication / + Add platform flow. If someone follows the main setup flow, they set the Redirect URI in step 5 and then skip straight to adding users, so they never see this guidance—even though that's where the platform type is chosen.

Maybe move this callout to step 5 instead? I don't think we need to repeat the full explanation in step 8, since that section already points back to the Redirect URI guidance. If we want a reminder there, a simple note like "Select Web as the platform type" should be enough.

>     When selecting the platform type, choose **Web**, not **Single-page application (SPA)**. The <Constant name="dbt" /> SSO integration is a confidential client that redeems the authorization code from the server using a client secret. If you add the **Redirect URI** under **SPA**, Entra ID enforces PKCE and rejects the server-side token exchange, causing sign-in to fail with the error `AADSTS9002325: Proof Key for Code Exchange is required for cross-origin authorization code redemption.`

Author: ```suggestion
    When selecting the platform type, choose **Web**, not **Single-page application (SPA)**. The <Constant name="dbt" /> SSO integration redeems the authorization code from the server using a client secret. So, if you add the **Redirect URI** under **SPA**, Entra ID enforces PKCE and rejects the server-side token exchange, causing sign-in to fail with the error `AADSTS9002325: Proof Key for Code Exchange is required for cross-origin authorization code redemption.`
```
